### ISSUE TITLE
Portable version for Mac : no bin directory

### ISSUE THREAD

**Issue by mlep** [JabRef/user-documentation#291]
In https://docs.jabref.org/installation , section Portable version, it is stated that:
`Inside the archive files you will find a bin subdirectory which contains the binary needed to run JabRef (i.e., JabRefMain for Linux and MacOS,[...]`.
However, in JabRef-5.0-portable_macos.tar.gz , there is no directory `bin`. There is a directory `MacOS` containing a file named `JabRef`.
Not having a Mac, I am not sure about the proper directory for Mac

**Comment by Siedlerchr** [comment]
The JabRef.app is also a kind of directory (also a kind of zip file) You can open it with a file manager.

**Comment by mlep** [comment]
So, it it the file `JabRef.app/Contents/MacOS/Jabref`?

**Comment by Siedlerchr** [comment]
Sorry,  I misunderstood your question. 
The JabRef.app is the "binary"/exe file as well. On macOS you simply double click the .app file and JabRef starts.
Even the dmg is trechnically under the hood a portable variant. only the pkg is a real installer.

**Comment by koppor** [comment]
Is this solved in #290?

```markdown
for Mac, this is the file `JabRef.app/Contents/MacOS/Jabref`
```

**Comment by mlep** [comment]
I do not think #290 is solved: the more it goes, the less clear it is to me... :sob: 
- for Windows, shall we advertise the use of `JabRef\runtime\bin\JabRef.bat` or of `JabRef\JabRef.exe`? My understanding is that Windows' users will, by default, click on the first file with .exe extension they see. :facepalm: 
- for Mac, my understanding is that launching the file `JabRef.app/Contents/MacOS/Jabref` is not what users do (see above comment by @Siedlerchr). But I am not a Mac user... @Siedlerchr : since you are talking Apple Mac language, please, could you rephrase the dedicated sentence at https://docs.jabref.org/installation?

**Comment by Siedlerchr** [comment]
@mlep  I adapted the mac part. 
The windows is currently apparently an issue with the jdk15, I'm currently doing a build with jdk14 again.
Exe should be fine.
 https://github.com/JabRef/jabref/pull/6802/checks?check_run_id=1037361296

**Comment by mlep** [comment]
Thanks. I have modified the info for windows (direct commit.
Issue closed! Thanks!
